src/System.Device.Gpio/System/Device/I2c/Devices/UnixI2cDevice.Linux.cs

```diff
@@ -5,12 +5,12 @@
 using System.IO;
 using System.Runtime.InteropServices;
 
-namespace System.Device.I2c.Devices
+namespace System.Device.I2c
 {
     /// <summary>
     /// Represents an I2C communication channel running on Unix.
     /// </summary>
-    public class UnixI2cDevice : I2cDevice
+    internal class UnixI2cDevice : I2cDevice
```

Comment:
We don't need an internal class here. Instead what we need is simply to move all of this code under I2CDevice.Linux.cs. Sorry if that didn't make sense before. Ideally what we want is not to need to have extra internal classes to shell out to, instead we just want to have one I2CDevice class, which in Linux has the copied implementation from the previous UnixI2CDevice, and in windows it uses what used to be Windows10I2cDevice. Does that make sense?

Reply:
Sure.  Lemme review.  Man, y'all are making sweat here 🏃 😓

Reply:
@shaggygi don't do anything yet, we're talking offline about this right now

Reply:
Hey @shaggygi sorry for randomizing you, So I think we want to talk a bit more about the comment I gave and maybe have a small API review for it. In the meantime, I'm fine with taking this as is, except that we should fix the CI Build, I'll let you know how to do that.

Reply:
@joperezr no prob.  i was expecting some tap dancing with this breaking change 😄 i'll check back in a few.